cli/auth_nkey_command.go

```diff
@@ -256,3 +272,106 @@ func (c *authNKCommand) verifyAction(_ *fisk.ParseContext) error {
 
 	return nil
 }
+func (c *authNKCommand) sealAction(_ *fisk.ParseContext) error {
+	var err error
+	var kp nkeys.KeyPair
+
+	keyData, err := c.readKeyFile(c.keyFile)
+	if err != nil {
+		return err
+	}
+
+	// try it as public, then as seed
+	kp, err = nkeys.FromPublicKey(string(keyData))
+	if errors.Is(err, nkeys.ErrInvalidPublicKey) {
+		kp, err = nkeys.FromSeed(keyData)
+	}
+	if err != nil {
+		return err
+	}
+
+	content, err := os.ReadFile(c.dataFile)
+	if err != nil {
+		return err
+	}
+
+	if nkeys.IsValidPublicCurveKey(c.counterpartKey) {
+		encryptedData, err := kp.Seal(content, c.counterpartKey)
+		if err != nil {
+			return err
+		}
+		if c.useB64 {
+			encryptedData = []byte(base64.StdEncoding.EncodeToString(encryptedData))
+		}
+
+		if c.outFile != "" {
+			f, err := os.Create(c.outFile)
+			if err != nil {
+				return err
+			}
+			_, err = f.Write(encryptedData)
+			if err != nil {
+				return err
+			}
+			return nil
+		} else {
+			fmt.Println(string(encryptedData))
+			return nil
+		}
+	}
+
+	return errors.New("failed to seal message")
+}
+func (c *authNKCommand) openAction(_ *fisk.ParseContext) error {
```

Comment:
rename to `unsealAction` please and then I guess the general style comments above would apply here also as they are roughly same layout.

Otherwise this is great stuff thanks for the contrib